src/components/app/main/main.js

```diff
@@ -2,16 +2,18 @@
 
 import React from 'react';
 
-import './pageContainer.css';
+import './main.css';
 
 /**
  * A presentational component for rendering a page.
  * The children of the component should contain the entire page layout.
  */
-export default function PageContainer(props) {
-    return (
-        <div className="page-container">
-            {props.children}
-        </div>
-    );
-}
+const Main = (props) => (
+  <main className="app-main">
+    <div className="main-container">
+      { props.children }
```

Comment:
Is props.children a list or object? The reason i ask is because if it is just single object then props.child might be better name

Reply:
It's a react property. https://reactjs.org/docs/react-api.html#reactchildren